src/plugins/discover/public/application/components/discover_grid/get_render_cell_value.tsx

```diff
@@ -108,8 +109,7 @@ export const getRenderCellValueFn = (
 
   if (field && field.type === '_source') {
     if (isDetails) {
-      // nicely formatted JSON for the expanded view
-      return <span>{JSON.stringify(row, null, 2)}</span>;
+      return <JsonCodeEditor hit={row} hasLineNumbers={false} width={370} />;
```

Comment:
Could you please provide some details why `370` and maybe create constant for this?